docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,248 @@ The following table shows how risk levels are applied to a host, based on the no
 
 |==============================================
 
+[[enable-host-risk-score]]
 [discrete]
-=== Deploy host risk score
+=== Enable host risk score
 
-NOTE: To enable the host risk score feature, you must have alerts in your environment. If you have none, the *Enable via Dev Tools* button will be greyed out.
+NOTE: To enable the host risk score feature, you must have alerts in your environment. If you previously enabled host risk score and are upgrading the {stack} to 8.5 or newer, refer to <<upgrade-host-risk-score>> in this topic. 
 
-To deploy the host risk score framework in your environment:
+You can enable host risk score from the following places in the  {security-app}:
+
+* The Entity Analytics dashboard 
+* The *Host risk* tab on the Hosts page
+* The *Host risk* tab on a host's details page
+
+Or, in {kib}, you can enable host risk score in Console. 
```

Comment:
@angorayc @SourinPaul  Do we still want to point users to the Dev Tools installation?

Reply:
@ajosh0504 Only keep this as a reference in the documentation for 8.5. If I recall, @angorayc suggestion was to retain this as a backup mechanism given the dashboard enablement is just introduced and we dont have any data from real-life enablement.

I think there is no harm in retaining it unless we are confident of the onboarding process and then remove it from the documentation as well.

Reply:
I remember in our previous discussion that we wanted to keep enable via dev tools in the doc, but I'm totally agree if we want to remove it in the future.